Dockerfile

```diff
@@ -1,5 +1,29 @@
-FROM jrei/systemd-debian:12
-RUN apt update && apt install -y sudo wget procps curl systemd iproute2 && rm -rf /var/lib/apt/lists/*
+FROM debian:bookworm
```

Comment:
Maybe we add a comment linking to where the logic originated from?

The repo's license file: https://github.com/j8r/dockerfiles/blob/master/LICENSE#L5

Reply:
Hm, doesn't matter, because this was crated from scratch, but if this already authored. That's ok to create link.